pkg/plugin/settings.go

```diff
@@ -186,6 +187,19 @@ func LoadSettings(ctx context.Context, config backend.DataSourceInstanceSettings
 		}
 	}
 
+	settings.EnableRowLimit = false
+	if jsonData["enableRowLimit"] != nil {
+		if enableRowLimitString, ok := jsonData["enableRowLimit"].(string); ok {
+			settings.EnableRowLimit, err = strconv.ParseBool(enableRowLimitString)
+			if err != nil {
+				backend.Logger.Warn("Failed to parse enableRowLimit value, defaulting to false", "error", err)
+				settings.EnableRowLimit = false
```

Comment:
`settings.EnableRowLimit` doesn't need to be explicitly set to false on 190, as bool type zero value is false.

also, `strconv.ParseBool()` returns `false` in the error [case](https://cs.opensource.google/go/go/+/refs/tags/go1.24.4:src/strconv/atob.go;l=17), so we don't need 196 either.

Reply:
Updated